api/server/router/volume/volume.go

```diff
@@ -44,6 +44,14 @@ func (r *router) initRoutes() {
 			handlers.NewSchemaValidator(nil, schema.ServiceVolumeMapSchema, nil),
 		),
 
+		// get all volumes from all services with attachments
+		httputils.NewGetRoute(
```

Comment:
The post handler to which you referred was for a different route -- the actual attach/detach routes. The get routes, `volumes` and `volumesForService` shouldn't need post args handlers as they don't have post bodies. Did you mean the attach/detach routes when you mentioned the post args handler? I just want to make sure page==same.